### DISCUSSION TITLE
Need Help Loading Map on OsmAnd

### DISCUSSION THREAD

**Discussion by JackZ295** [osmandapp/OsmAnd#13393] [Q&A]
Hi! I tried to download a map on OsmAnd, but I cannot seem to get it to load. Here was how I went about doing so: 

1. I pressed the 3 line icon on the lower left after opening the app. 
2. I tapped on "Download Maps" 
3. I tapped on Central America. 
4. I then tapped on Caribbean archipelago
5. I tapped on Standard map. 
6. The standard map downloaded, but there was no option to load it. No map loaded on the app. When I clicked on "Standard Map," the only options I had were to either to download the map again or to remove it. 

Does anyone have any input regarding this? Thanks!

**Comment by karl-01** [comment]
There is no need for activation of maps.
As far as I understand your description, the map is now active and you should be able to see it.
Try to zoom out, may be you find the map....
